### DISCUSSION TITLE
Can units be added on Y-axis?

### DISCUSSION THREAD

**Discussion by sanshigongA** [eclipse-birt/birt#1757] [Q&A]
<img width="640" alt="微信图片_20240625150328" src="https://github.com/eclipse-birt/birt/assets/107231757/0caaeb0a-247f-47c7-8df0-bd01ea67b42e">

**Comment by speckyspooky** [comment]
May be you can use the axis-title to get the units, e.g.:

![grafik](https://github.com/eclipse-birt/birt/assets/41593722/0d44cf71-e1ea-4f3d-a0f3-2a0b425b5503)

Is it not the same like in your screen but an option.

The rest is more complicate to change the axis and in your case to add aditional text and move it above the axis.
